react/features/base/participants/functions.js

```diff
@@ -145,17 +145,10 @@ export function getParticipantCountWithFake(stateful: Object | Function) {
  * FIXME: Remove the hardcoded strings once interfaceConfig is stored in redux
  * and merge with a similarly named method in {@code conference.js}.
  *
- * @param {(Function|Object)} stateful - The (whole) redux state, or redux's
- * {@code getState} function to be used to retrieve the state.
- * @param {string} id - The ID of the participant's display name to retrieve.
- * @private
+ * @param {Object} participant - Participant object.
  * @returns {string}
  */
-export function getParticipantDisplayName(
-        stateful: Object | Function,
-        id: string) {
-    const participant = getParticipantById(stateful, id);
-
+export function getParticipantDisplayName(participant: Object) {
```

Comment:
I don't think this is good going forward. `interfaceConfig` will eventually be part of the state, so how would we fetch stuff from it, if we don't have access to it?

Reply:
Well, currently we don't need the state object in those calls. Would you rather refactor these to get the participant from the state instead of using the passed value only for the sake of having the state object available there for some future use? I wouldn't, but I'm curious what you think.

Reply:
I'd have a function which takes `state` and `participant`. It uses each as needed. Now we are not using state and we know this, but we will, so let's not put ourselves in a position where we need to do it again later...